druid/src/widget/svg.rs

```diff
@@ -1,568 +1,195 @@
-// Copyright 2019 The Druid Authors.
```

Comment:
Confused by this '- }' in the diff, since it doesn't appear in the original file (wondering if github diff interface is mangling it).

Otherwise, could you explain the removal of the copyright?

Reply:
this is a widget I implemented in my own codebase to get essentially fully functioning SVG rendering. vs the hodge podge druid currently has. its meant as an example implementation for druid to consider. just slapped it in place.

benefits:
1. correctly renders essentially all svgs
2. removal of the font loading for every svg dramatically speeds up rendering.
3. simple caching to prevent unnecessary re-rendering.
4. no svg rendering code is owned by druid.

downsides:
1. dependency on tiny-skia.

Reply:
Marking as resolved, I no longer see the diff weirdness (Which was in the in-tree sources not matching what was actually in tree), and your latest push leaves copyright header intact thanks.